fdbserver/GlobalTagThrottler.actor.cpp

```diff
@@ -494,8 +458,8 @@ class GlobalTagThrottlerImpl {
 		}
 		return result;
 	}
-	uint32_t busyReadTagCount() const { return lastBusyReadTagCount; }
-	uint32_t busyWriteTagCount() const { return lastBusyWriteTagCount; }
+	uint32_t busyReadTagCount() const { return lastBusyTagCount; }
+	uint32_t busyWriteTagCount() const { return lastBusyTagCount; }
```

Comment:
Is there value in keeping these as two separate functions, and are the names still accurate?

Reply:
The reason for this naming is that these are virtual methods of the `ITagThrottler` interface. For the other `TagThrottler` implementation the distinction is meaningful, but for `GlobalTagThrottler` all busy tags count as both read-busy and write-busy.